rio/src/RIO/List/Partial.hs

```diff
@@ -17,15 +17,20 @@ module RIO.List.Partial
   , Data.List.maximumBy
   , Data.List.minimumBy
 
-  -- * Building lists
-
   -- ** Scans
-  , Data.List.scanl1
-  , Data.List.scanr1
+  , RIO.List.Partial.scanl1
```

Comment:
This is actually still a breaking change, since now the identifier can conflict with one imported from elsewhere (like the one in `RIO.List`, meaning this will almost certainly break code).

Reply:
🤦‍♂️